lib/gettext.js

```diff
@@ -8,8 +8,7 @@
     var defaults = {
       domain: 'messages',
       locale: document.documentElement.getAttribute('lang') || 'en',
-      plural_form: 'nplurals=2; plural=(n!=1);',
-      plural_func: function (n) { return { nplurals: 2, plural: (n!=1) ? 1 : 0 }; },
+      plural_func: function (n) { return { nplural: 2, plural: (n!=1) ? 1 : 0 }; },
```

Comment:
Because in other code it calls nplural, not nplurals
see line 67: return new Function("n", 'var plural, nplurals; '+ plural_form +' return { **nplural**: nplurals, ...
and line 94: if ('undefined' === typeof plural.plural || plural.plural > plural.**nplural**